### DISCUSSION TITLE
Carlinkit 5.0 bricked? Firmware?

### DISCUSSION THREAD

**Discussion by Levonson01** [ludwig-v/wireless-carplay-dongle-reverse-engineering#182] [Q&A]
Hey guys,

so recently my Carlinkit 5.0 stopped connection with my phone. I dont see it in my Bluetooth nor Wlan list. 

When i start the device, there is a blinking yellowish light. I tried flashing the firmware .img.

It started blinking green and red while flashing. After the flash completed it didnt was a solid green or red as expected. It was blinking green only forever. When disconnected it seems like nothing changed and the old blinking yellowish color is back after start up.

What can i do? Do you maybe have an alternative firmware for resetting the device. I don't think its bricked cause the LED‘s still work..

Thank you for your help!

**Comment by imedium3** [comment]
Hey Levonson01,

That's what happened with me exactly, try flashing the device using this custom firmware 

https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/tree/master/Custom_Firmware/U2AC/_AUTOKIT/2024.01.18.1053

Wait until the led flashing green then reboot and disconnect the usb.
remove the firmware from the usb then once it is flashing Yellow insert the empty usb you will get a log
Then check it i think the chip id is wrongly detected as per my case.
